modules/openapi-generator/src/main/resources/csharp-netcore/libraries/httpclient/ApiClient.mustache

```diff
@@ -189,30 +187,28 @@ namespace {{packageName}}.Client
         /// <summary>
         /// Initializes a new instance of the <see cref="ApiClient" />, defaulting to the global configurations' base url.
         /// </summary>
-        public ApiClient()
+        public ApiClient(HttpClient client = null, HttpClientHandler handler = null) : this({{packageName}}.Client.GlobalConfiguration.Instance.BasePath, client, handler)
         {
-            _baseUrl = {{packageName}}.Client.GlobalConfiguration.Instance.BasePath;
-    {{#reUseHttpClient}}
-            _httpClientHandler = new HttpClientHandler();
-            _httpClient = new HttpClient(_httpClientHandler);
-    {{/reUseHttpClient}}
         }
 
         /// <summary>
-        /// Initializes a new instance of the <see cref="ApiClient" />
+            /// Initializes a new instance of the <see cref="ApiClient" />
```

Comment:
Minor suggestion: remove these spaces

Reply:
Will do. I'll also maybe make the HttpClient private again on the class as it is now injected and doesn't need to be modified after creation anymore.